nix/patches/ghc-8.0.2-darwin-rec-link.patch

```diff
@@ -2,23 +2,12 @@ diff --git a/compiler/main/DriverPipeline.hs b/compiler/main/DriverPipeline.hs
 index acd0d61..3e83c15 100644
 --- a/compiler/main/DriverPipeline.hs
 +++ b/compiler/main/DriverPipeline.hs
-@@ -1916,6 +1916,7 @@ linkBinary' staticLink dflags o_files dep_packages = do
+@@ -1916,5 +1916,5 @@ linkBinary' staticLink dflags o_files dep_packages = do
                        ++ pkg_framework_opts
                        ++ debug_opts
                        ++ thread_opts
+-                      ++ (if (platformOS platform `elem` [OSDarwin]) then [ "-Wl,-dead_strip_dylibs" ] else [])
 +                      ++ (if (platformOS platform `elem` [OSDarwin, OSiOS]) then [ "-Wl,-dead_strip_dylibs", "-Wl,-dead_strip" ] else [])
```

Comment:
Is this a patch for ghc 8.4.4? Can we rename the file?

Reply:
this is not really 8.4.4 specific. It's just a more comprehensive patch than what nixpkgs upstream does.